packages/cloud-cognitive/src/components/Datagrid/styles/addons/_FilterLeftPanel.scss

```diff
@@ -0,0 +1,122 @@
+/*
+* Licensed Materials - Property of IBM
+* 5724-Q36
+* (c) Copyright IBM Corp. 2022
+* US Government Users Restricted Rights - Use, duplication or disclosure
+* restricted by GSA ADP Schedule Contract with IBM Corp.
+*/
+
+@import '../variables';
+
+.#{$block-class}-filter-left-panel__container {
+  position: relative;
+  width: rem(320px);
+  height: max(var(--filter-panel-min-height), 100vh);
+  border-top: 1px $ui-03 solid;
+  border-right: 1px $ui-03 solid;
```

Comment:
I think this should actually be added to the open class only since it’ll appear even when the container is collapsed.

![image](https://user-images.githubusercontent.com/1717398/217414320-f1ac505c-2947-41df-bdd3-817c2700dae9.png)

```scss
.#{$block-class}-filter-left-panel__container..#{$block-class}-filter-left-panel--open {
  border-right: 1px $ui-03 solid;
}
```